protocols/directfund/serde.go

```diff
@@ -24,7 +24,8 @@ type jsonObjective struct {
 // MarshalJSON returns a JSON representation of the DirectFundObjective
 //
 // NOTE: Marshal -> Unmarshal is a lossy process. All channel data
-//       (other than Id) from the field C is discarded
+//
+//	(other than Id) from the field C is discarded
```

Comment:
These changes were automatic, but they highlight something that seems off. Probably this line should never have been indented?

Reply:
Good catch! I've cleaned up the spacing in [094f166](https://github.com/statechannels/go-nitro/pull/821/commits/094f166feab9267d2520b9f8ca55206c8ae9e799)